pkgs/by-name/ke/keila/mix_deps.nix

```diff
@@ -0,0 +1,896 @@
+        pkg = "html_entities";
+        version = "${version}";
+        sha256 = "c53ba390403485615623b9531e97696f076ed415e8d8058b1dbaa28181f4fdcc";
+      };
+
+      beamDeps = [];
+    };
+
+    httpoison = buildMix rec {
+      name = "httpoison";
+      version = "1.8.0";
+
+      src = fetchHex {
+        pkg = "httpoison";
+        version = "${version}";
+        sha256 = "28089eaa98cf90c66265b6b5ad87c59a3729bea2e74e9d08f9b51eb9729b3c3a";
+      };
+
+      beamDeps = [ hackney ];
+    };
+
+    idna = buildRebar3 rec {
+      name = "idna";
+      version = "6.1.1";
+
+      src = fetchHex {
+        pkg = "idna";
+        version = "${version}";
+        sha256 = "92376eb7894412ed19ac475e4a86f7b413c1b9fbb5bd16dccd57934157944cea";
+      };
+
+      beamDeps = [ unicode_util_compat ];
+    };
+
+    jason = buildMix rec {
+      name = "jason";
+      version = "1.4.4";
+
+      src = fetchHex {
+        pkg = "jason";
+        version = "${version}";
+        sha256 = "c5eb0cab91f094599f94d55bc63409236a8ec69a21a67814529e8d5f6cc90b3b";
+      };
+
+      beamDeps = [ decimal ];
+    };
+
+    makeup = buildMix rec {
+      name = "makeup";
+      version = "1.1.0";
+
+      src = fetchHex {
+        pkg = "makeup";
+        version = "${version}";
+        sha256 = "0a45ed501f4a8897f580eabf99a2e5234ea3e75a4373c8a52824f6e873be57a6";
+      };
+
+      beamDeps = [ nimble_parsec ];
+    };
+
+    makeup_elixir = buildMix rec {
+      name = "makeup_elixir";
+      version = "0.16.0";
+
+      src = fetchHex {
+        pkg = "makeup_elixir";
+        version = "${version}";
+        sha256 = "28b2cbdc13960a46ae9a8858c4bebdec3c9a6d7b4b9e7f4ed1502f8159f338e7";
+      };
+
+      beamDeps = [ makeup nimble_parsec ];
+    };
+
+    makeup_erlang = buildMix rec {
+      name = "makeup_erlang";
+      version = "0.1.1";
+
+      src = fetchHex {
+        pkg = "makeup_erlang";
+        version = "${version}";
+        sha256 = "174d0809e98a4ef0b3309256cbf97101c6ec01c4ab0b23e926a9e17df2077cbb";
+      };
+
+      beamDeps = [ makeup ];
+    };
+
+    metrics = buildRebar3 rec {
+      name = "metrics";
+      version = "1.0.1";
+
+      src = fetchHex {
+        pkg = "metrics";
+        version = "${version}";
+        sha256 = "69b09adddc4f74a40716ae54d140f93beb0fb8978d8636eaded0c31b6f099f16";
+      };
+
+      beamDeps = [];
+    };
+
+    mime = buildMix rec {
+      name = "mime";
+      version = "2.0.5";
+
+      src = fetchHex {
+        pkg = "mime";
+        version = "${version}";
+        sha256 = "da0d64a365c45bc9935cc5c8a7fc5e49a0e0f9932a761c55d6c52b142780a05c";
+      };
+
+      beamDeps = [];
+    };
+
+    mimerl = buildRebar3 rec {
+      name = "mimerl";
+      version = "1.3.0";
+
+      src = fetchHex {
+        pkg = "mimerl";
+        version = "${version}";
+        sha256 = "a1e15a50d1887217de95f0b9b0793e32853f7c258a5cd227650889b38839fe9d";
+      };
+
+      beamDeps = [];
+    };
+
+    mjml = buildMix rec {
```

Comment:
Currently fails to build with
```
== Compilation error in file lib/mjml/native.ex ==
** (MatchError) no match of right hand side value: {:error, :enoent}
    (rustler_precompiled 0.7.2) lib/rustler_precompiled.ex:1013: RustlerPrecompiled.write_metadata/2
    (rustler_precompiled 0.7.2) lib/rustler_precompiled.ex:209: RustlerPrecompiled.__using__/2
    lib/mjml/native.ex:32: (module)
```

Reply:
Yea, I might just revert to the version before the introduction of this mjml/rustler stuff. I looked a bit into it, and rustler wants to network access, and maybe that's why it's failing.